doc/content/reference/application-packages/lora-cloud-device-management-v1.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "LoRa Cloud Device Management v1"
+description: ""
+weight: 2
+---
+
+
+The LoRa Cloud Device Management v1 application package communicates the uplinks received from a DM-compatible device to the LoRa Cloud Device Management Service, and schedules the downlinks received from the service back to the device.
```

Comment:
What is a "DM-compatible device"?

Reply:
Device Management compatible device - wording taken from their official [docs](https://www.loracloud.com/documentation/device_management?url=overview.html). I've rewritten this to only `compatible`.